README.md

```diff
@@ -5,7 +5,7 @@ See documentation [here](doc/00-overview.md)
 ``` yaml
 use-extension:
   "@autorest/python": "5.0.0-dev.20200211.1"
-  "@autorest/clicommon": "0.3.7"
+  "@autorest/clicommon": "0.3.8"
   #"@autorest/python": "latest"
   

```

Comment:
could you remove 'scope-clicommon: false' in the readme? now clicommon will only output file when --debug is marked